spec/book_spec.rb

```diff
@@ -0,0 +1,19 @@
+# book_spec.rb
```

Comment:
- [ ] Great work on making the tests but I guess you missed one file 😄 which is a nameable.rb file 🤔 Kindly add unit tests for that file as well to make your app well tested as suggested by the previous reviewer 👍